cdap-docs/_common/_themes/cdap/static/cdap.css_t

```diff
@@ -214,17 +200,34 @@ div.body h4 { background-color: #EDEDED; margin: 30px 0px 10px 0px; }
 div.body h5 { background-color: #F2F2F2; margin: 30px 0px 10px 0px; }
 div.body h6 { background-color: #F2F2F2; }
 
+a {
+    color: #ff6600;
+    text-decoration: none;
+}
+
+a.current {
+    font-weight: bold;
+}
+
+a:hover {
+    color: #005B81;
+    text-decoration: underline;
```

Comment:
Isn't this the default behavior by the browser? To underline on hover? Any specific reason why we have `text-decoration` to `none` in `a{...}` selector and set to `underline` here?